corehq/apps/hqadmin/views.py

```diff
@@ -561,6 +561,24 @@ def _lookup_id_in_couch(doc_id, db_name=None):
 
 
 @require_superuser
+def web_user_lookup(request):
+    template = "hqadmin/web_user_lookup.html"
+    set_bootstrap_version3()
+    web_user_email = request.GET.get("q")
+    if not web_user_email:
+        return render(request, template, {})
+
+    web_user = WebUser.get_by_username(web_user_email)
+    if web_user is None:
+        messages.error(
+            request, "Sorry, no user found with email {}. Did you enter it correctly?".format(web_user_email)
```

Comment:
Makes sense, thanks. 
I would also argue that this isn't necessarily an error condition, and we should show the fact that no users were found in the table (as we do when filters come up with no results), but for an internal facing page probably doesn't matter that much.